docs/tools/curl.md

```diff
@@ -9,6 +9,10 @@
 Also, you can download executable from their website and run it from a directory or make it discoverable through `PATH`,
 so it can be run from anywhere. Using a package manager is preferable as it simplifies the process of installing and updating the tool.
 
+Windows and MacOS come with outdated versions of curl. After installing curl, run `curl --version` (or `curl.exe --version` in
```

Comment:
Using newest versions is not very mandatory, curl API haven't changed much in years, but keeping it updated is good idea. Plus it updates with system.